src/connectors/hits/connectHits.js

```diff
@@ -77,8 +79,14 @@ export default function connectHits(renderFn, unmountFn) {
     },
 
     render({ results, instantSearchInstance }) {
-      if (widgetParams.escapeHits && results.hits && results.hits.length > 0) {
-        results.hits = escapeHits(results.hits);
+      if (results.hits && results.hits.length > 0) {
+        if (typeof widgetParams.transformItems === 'function') {
```

Comment:
Why not provide a default implementation (identity) like in other widget and remove this if statement? Also I'd suggest to test the type sooner at the initialization.

Reply:
Glad you caught that. I wasn't sure if I should have updated the code flow of the connector or should have adapted my code on the existing flow.